pom.xml

```diff
@@ -159,5 +159,30 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
in my point of view, we don't need this profile, the jib push profile was created to make it easy to push the image, but if you want to build locally you can simply run `mvn clean package jib:dockerBuild`. Can you remove this profile in favour of a simple readme.md update with this command for the local build?